src/esi/Include.cc

```diff
@@ -335,9 +335,8 @@ ESIInclude::ESIInclude(esiTreeParentPtr aParent, int attrcount, char const **att
              */
             debugs(86, 5, "ESIIncludeNew: Requesting alternate '" << attr[i+1] << "'");
 
-            assert (alt.getRaw() == NULL); /* TODO: FIXME */
+            assert(!alt.getRaw());
```

Comment:
If we start out of scope manual polishing, why stop midway?

```suggestion
            assert(!alt);
```

Reply:
Reverted in 6871c39